### ISSUE TITLE
Migration to 3.5.2 failing with `Error creating bean with name 'databaseUpdater':`

### ISSUE THREAD

**Issue by KesavanKing** [naver/ngrinder#690]
Hello,

Thank you so much for this nice tool. We have been using ngrinder as our primary load generator tool for a quite long time.
I observe some new  features getting delivered in every new releases.

We were running ngrinder as  kubernetes workloads with version 3.4.4 and when I try to migrate to new image of version 3.5.2, controller failing with following error logs.

```
2020-11-13 05:49:27,293 ERROR SpringApplication.java:837 : Application run failed
org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'databaseUpdater': Invocation of init method failed; nested exception is org.ngrinder.common.exception.NGrinderRuntimeException: Exception occurs while Liquibase update DB
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:160)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyBeanPostProcessorsBeforeInitialization(AbstractAutowireCapableBeanFactory.java:415)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1786)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:594)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:516)
	at org.springframework.beans.factory.support.AbstractBeanFactory.lambda$doGetBean$0(AbstractBeanFactory.java:324)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:226)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:322)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:311)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1109)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:869)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:551)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:143)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:758)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:750)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:315)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1237)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at org.ngrinder.starter.NGrinderControllerStarter.main(NGrinderControllerStarter.java:261)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:109)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
	at org.springframework.boot.loader.WarLauncher.main(WarLauncher.java:59)
Caused by: org.ngrinder.common.exception.NGrinderRuntimeException: Exception occurs while Liquibase update DB
	at org.ngrinder.common.util.ExceptionUtils.processException(ExceptionUtils.java:53)
	at org.ngrinder.infra.init.DatabaseUpdater.init(DatabaseUpdater.java:80)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleElement.invoke(InitDestroyAnnotationBeanPostProcessor.java:389)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMetadata.invokeInitMethods(InitDestroyAnnotationBeanPostProcessor.java:333)
	at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:157)
	... 29 common frames omitted
Caused by: liquibase.exception.LockException: liquibase.exception.DatabaseException: Error executing SQL UPDATE PUBLIC.DATABASECHANGELOGLOCK SET LOCKED = TRUE, LOCKEDBY = 'controller-new-0.controller-new.whitenoise.svc.cluster.local (100.96.2.15)', LOCKGRANTED = '2020-11-13 05:49:27.222' WHERE ID = 1 AND LOCKED = FALSE: Value too long for column "LOCKED CHAR(1) NOT NULL": "'TRUE' (4)"; SQL statement:
UPDATE PUBLIC.DATABASECHANGELOGLOCK SET LOCKED = TRUE, LOCKEDBY = 'controller-new-0.controller-new.whitenoise.svc.cluster.local (100.96.2.15)', LOCKGRANTED = '2020-11-13 05:49:27.218' WHERE ID = 1 AND LOCKED = FALSE [22001-197]
	at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:242)
	at liquibase.lockservice.StandardLockService.waitForLock(StandardLockService.java:170)
	at liquibase.Liquibase.update(Liquibase.java:196)
	at liquibase.Liquibase.update(Liquibase.java:192)
	at liquibase.Liquibase.update(Liquibase.java:188)
	at liquibase.Liquibase.update(Liquibase.java:181)
	at org.ngrinder.infra.init.DatabaseUpdater.init(DatabaseUpdater.java:78)
	at org.ngrinder.starter.NGrinderControllerStarter.main(NGrinderControllerStarter.java:261)
Caused by: liquibase.exception.DatabaseException: Error executing SQL UPDATE PUBLIC.DATABASECHANGELOGLOCK SET LOCKED = TRUE, LOCKEDBY = 'controller-new-0.controller-new.whitenoise.svc.cluster.local (100.96.2.15)', LOCKGRANTED = '2020-11-13 05:49:27.222' WHERE ID = 1 AND LOCKED = FALSE: Value too long for column "LOCKED CHAR(1) NOT NULL": "'TRUE' (4)"; SQL statement:
```

If I deploy new controllers I didn't face this issue. But if we do an update for existing controller we face this issue.
Any workarounds to migrate ?

Regards,
Kesavan

**Comment by donggyu04** [comment]
@KesavanKing 

Hi, It seems a liquibase issue. As the liquibase version bumped up, there were changes to the table fields used internally.

If you use H2, Please run `ALTER TABLE DATABASECHANGELOGLOCK ALTER COLUMN LOCKED char(5);
` in your database before running ngrinder 3.5.2

**Comment by silloy** [comment]
3.5.3 have the same question

**Comment by donggyu04** [comment]
Fixed by #788
